sbol_utilities/sbol3_genbank_conversion.py

```diff
@@ -0,0 +1,43 @@
+import os
+import sbol3
+sbol3.set_namespace("https://testing.sbol3.genbank/")
+from Bio import GenBank
+GENBANK_PARSER = GenBank.RecordParser()
+
+# Conversion Constants (NOTE: most are placeholding and temporary for now)
+SAMPLE_GB_FILE_1 = "BBa_J23101.gb"          # Simplest file in genbank consisting only of a singular sequence
+SAMPLE_SBOL3_FILE_1 = "BBa_J23101_from_genbank_to_sbol3_direct.nt"
+COMP_TYPES = [sbol3.SBO_DNA]                # Temporarily assuming only dna components to be dealt with in genbank files
+COMP_ROLES = [sbol3.SO_ENGINEERED_REGION]   # Temporarilty assuming components to only have the engineered_region role
+GENBANK_FILE_1 = os.path.join(os.path.abspath(os.path.curdir), \
```

Comment:
backslashes are not needed within parentheses. Please remove them here and elsewhere in this file.